include/uv.h

```diff
@@ -1392,6 +1392,14 @@ UV_EXTERN int uv_ip6_name(const struct sockaddr_in6* src, char* dst, size_t size
 UV_EXTERN int uv_inet_ntop(int af, const void* src, char* dst, size_t size);
 UV_EXTERN int uv_inet_pton(int af, const char* src, void* dst);
 
+#if defined(IF_NAMESIZE)
+# define UV_IF_NAMESIZE IF_NAMESIZE
+#elif defined(IFNAMSIZ)
+# define UV_IF_NAMESIZE IFNAMSIZ
+#endif
```

Comment:
If neither of them are defined, then presumably the underlying OS does not support `if_indextoname` either.  Should that invoke an `#error` or `#warning`?

For the next version I'll place there a `#warning`, but of course that can be removed, if needed.